codelabs/gke-base/install/manifests.yml

```diff
@@ -42,7 +42,7 @@ spec:
     spec:
       containers:
       - name: halyard-daemon
-        image: gcr.io/spinnaker-marketplace/halyard:stable
+        image: gcr.io/spinnaker-marketplace/halyard@sha256:5c34d37b7e6fbcc1776d2cd9385ebaf063a7b6d670abacb0fa6c2b5886e5d47f
```

Comment:
Intentional to pin this?

Reply:
At the time, it was the version that had the fix we needed. Looks like that version is now also marked `stable`. Fixed.